wcfsetup/setup/db/install.sql

```diff
@@ -1541,7 +1541,8 @@ CREATE TABLE wcf1_user_avatar (
 	width SMALLINT(5) NOT NULL DEFAULT 0,
 	height SMALLINT(5) NOT NULL DEFAULT 0,
 	userID INT(10),
-	fileHash VARCHAR(40) NOT NULL DEFAULT ''
+	fileHash VARCHAR(40) NOT NULL DEFAULT '',
+    hasWebP TINYINT(1) NOT NULL DEFAULT 0
```

Comment:
Add this change to https://github.com/WoltLab/WCF/blob/master/wcfsetup/install/files/acp/update_com.woltlab.wcf_5.4_db.php, please.